lib/screens/settings.dart

```diff
@@ -19,45 +21,78 @@ class _SettingsScreenState extends State<SettingsScreen> {
   Widget build(BuildContext context) {
     return Container(
         color: Colors.white,
-        padding: EdgeInsets.all(50),
-        child: Align(
-            alignment: Alignment.center,
-            child: Column(
+        padding: EdgeInsets.only(top: 48),
+        child: Column (
+            children: <Widget> [
+              Align (
+                alignment: Alignment.center,
+              child: Column (
               mainAxisAlignment: MainAxisAlignment.center,
+              crossAxisAlignment: CrossAxisAlignment.center,
               children: <Widget>[
                 ClipOval(
                     child: Image.network(imageUrl,
                         width: 100, height: 100, fit: BoxFit.cover)),
                 SizedBox(height: 20),
-                Text('Welcome, this is the Settings Screen,',
-                    textAlign: TextAlign.center),
-                Text(displayName,
+                Text( displayName,
                     textAlign: TextAlign.center,
+                    
                     style:
-                        TextStyle(fontWeight: FontWeight.bold, fontSize: 25)),
-                SizedBox(height: 20),
-                FlatButton(
-                    shape: RoundedRectangleBorder(
-                      borderRadius: BorderRadius.circular(20),
-                    ),
-                    onPressed: () {
-                      widget._googleSignIn.signOut();
-                      Navigator.pop(context, false);
-                    },
-                    color: Colors.redAccent,
-                    child: Padding(
-                        padding: EdgeInsets.all(10),
-                        child: Row(
-                          mainAxisAlignment: MainAxisAlignment.center,
-                          crossAxisAlignment: CrossAxisAlignment.center,
-                          children: <Widget>[
-                            Icon(Icons.exit_to_app, color: Colors.white),
-                            SizedBox(width: 10),
-                            Text('Log out of Google',
-                                style: TextStyle(color: Colors.white))
-                          ],
-                        )))
+                        TextStyle(fontWeight: FontWeight.bold,  fontSize: 16)),
+                Padding (padding: EdgeInsets.all(8),),
```

Comment:
Paddings are not used as space in between containers put this as a parent of a sized box or the text widget instead